### PR TITLE
Release v4.1.1

### PR DESCRIPTION
**Pull request by parkhomenko**
Bumped to a new version: 4.1.1

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -24,9 +24,6 @@ Compatibility
 
 The following versions of Python are supported:
 
-* Python 2.7
-* Python 3.3
-* Python 3.4
 * Python 3.5
 * Python 3.6
 * Python 3.7
```
Comment (faunaee) [inline README.rst:29]: The Circle CI config adds support for Python 3.9. Do we support it? If so, shouldn't we list it here, and what about 3.8?

**Review by faunaee — COMMENTED** [review]
A question about supported Python versions.

File: README.rst (hunk 1 above)
Comment (macnealefauna) [reply README.rst:29]: As discussed in slack, let's add 3.8 and 3.9 if we have verified that tests pass, we should update the supported version here.

Comment (parkhomenko) [reply README.rst:29]: added

**Review by fireridlle — APPROVED** [review]

**Pull request merged**